keps/sig-network/1453-ingress-api/README.md

```diff
@@ -0,0 +1,1371 @@
+    implementation's controller.
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  At the high-level this usually will be in the form of "high percentile of SLI
+  per day <= X". It's impossible to provide a comprehensive guidance, but at the very
+  high level (they needs more precise definitions) those may be things like:
+  - N/A
+
+* **Are there any missing metrics that would be useful to have to improve
+  observability if this feature?**
+  Describe the metrics themselves and the reason they weren't added (e.g. cost,
+  implementation difficulties, etc.).
+  - N/A
+
+### Dependencies
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **Does this feature depend on any specific services running in the cluster?**
+  Think about both cluster-level services (e.g. metrics-server) as well
+  as node-level agents (e.g. specific version of CRI). Focus on external or
+  optional services that are needed. For example, if this feature depends on
+  a cloud provider API, or upon an external software-defined storage or network
+  control plane.
+
+  For each of the fill in the following, thinking both about running user workloads
+  and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - CloudProvider / Ingress Controller
+    - Usage description:
+      - In order for newer features to be utilized like PathType and IngressClass, the 
+      Ingress controller will need to be updated to support these fields and respond 
+      accordingly.
+
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirms the
+previous answers based on experience in the field._
+
+* **Will enabling / using this feature result in any new API calls?**
+  - N/A
+
+* **Will enabling / using this feature result in introducing new API types?**
+  - New networking.k8s.io types
+    - Ingress.Spec.Backend.Resource 
+    - Ingress.Spec.IngressClass    
+    - Ingress.Spec.Rules.HTTP.PathType
+  - These new types will vary in size depending on the implementation controller and overall size of the cluster
+
+* **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+  - Cloud providers will need to update their specific Ingress controller if 
+    they provide one as part of their offering.
+
+* **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+  Describe them providing:
+  - New networking.k8s.io types
+      - Ingress.Spec.Backend.Resource 
+      - Ingress.Spec.IngressClass    
+      - Ingress.Spec.Rules.HTTP.PathType
+  - These new types will vary in size depending on the implementation controller and overall size of the cluster
```

Comment:
I think "size of the cluster" doesn't really matter on the size of Ingress object, right?

[What I meant previously was that number of Ingress objects that you may be able to create can depend on cluster size. But here it should't matter unless i'm missing something. So let's remove that part of the sentence]